sdk/maps/Azure.Maps.Rendering/src/Models/ImagePathStyle.cs

```diff
@@ -25,7 +25,7 @@ public class ImagePathStyle
         public ImagePathStyle(IList<GeoPosition> pathPositions)
         {
             Argument.AssertNotNull(pathPositions, nameof(pathPositions));
-            if (pathPositions?.Count == 0)
+            if (pathPositions == null || pathPositions.Count == 0)
```

Comment:
@dubiety:  This change does look to fix a logic error in Maps.   Can you give it a look, please?